recipes/r-tfplot/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set version = '2021.6-1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tfplot
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/tfplot_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/tfplot/tfplot_{{ version }}.tar.gz
+  sha256: acd36cc54dda3f9bbd90a03460c58ef8116f10d842cfcdfb3689ca4d454b1570
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-tframe
+  run:
+    - r-base
+    - r-tframe
+
+test:
+  commands:
+    - $R -e "library('tfplot')"           # [not win]
+    - "\"%R%\" -e \"library('tfplot')\""  # [win]
+
+about:
+  home: http://tsanalysis.r-forge.r-project.org/
+  license: GPL-2.0-or-later
```

From CRAN meta-data `License: GPL-2`: 
```suggestion
  license: GPL-2.0-only
```